packages/html/lib/index.js

```diff
@@ -0,0 +1,87 @@
+const { extname } = require('path');
+
+const getFiles = (bundle) => {
```

Comment:
This function should ONLY get entry point files (isEntry: true) and ignore the rest. Reason:
- For IIFE and UMD formats, there will only be a single entry point anyway
- All other formats import their dependencies themselves. If you add all files as script tags, you will at least destroy dynamic lazy loading of chunks (if it works at all).

Reply:
Disagree here. Various assets can be added to a bundle's files via plugins, and we should pick those up. CSS is the major example here.

Reply:
This function puts all files .js files into an array and throws away all meta information. It is virtually impossible to distiguish  the auto-generated chunks from the entry points. This makes the plugin rather useless for code-splitting IMO. Either you provide some meta information, or you ignore the non-entry chunks. I would be very confused if a CSS plugin was broken if you ignored the auto-generated chunks, as they are still imported by the main JavaScript. And maybe this even breaks CSS plugins in so far as maybe their chunks are meant to be loaded dynamically together with their content while you make them load up-front instead.

Reply:
That makes the case for providing the bundle metadata to the `template` option function for complex builds that the default template doesn't suit.

Reply:
That would be sufficient

Reply:
With regard to my initial comment, maybe I mis-stated what I mean: This function should of course return all assets, but with regard to chunks, it should only return entry chunks, as all non-entry chunks (read: .js files) will be loaded automatically by the entry chunks, no additional work necessary. But additionally having the bundle available to the template still would not hurt.